apps/opencs/view/render/terrainstorage.hpp

```diff
@@ -15,12 +16,25 @@ namespace CSVRender
     {
     public:
         TerrainStorage(const CSMWorld::Data& data);
+        float mAlteredHeight[ESM::Land::LAND_SIZE * ESM::Land::LAND_SIZE + ESM::Land::LAND_SIZE];
```

Comment:
Why do you need extra `ESM::Land::LAND_SIZE`? For example [here](https://github.com/OpenMW/openmw/pull/2520/files#diff-2cbfa0350b79af9ac769a7924fa05cd8R53-R59) loop is only over `ESM::Land::LAND_SIZE * ESM::Land::LAND_SIZE` elements.